inst/json/metadata_0.1.0.json

```diff
@@ -42,6 +41,14 @@
         { "id": "medium", "label": "Medium" },
         { "id": "high", "label": "High" }
       ]
+    },
+    {
+      "id": "hospital_capacity",
+      "label": "Hospital capacity",
+      "parameterType": "numeric",
+      "ordered": false,
+      "step": 100,
+      "updateNumericFrom": null
```

Comment:
Since there are two ways of talking about hospital capacity - per 100k and absolute - can we document somewhere which one we are using in this output? Seems like a job for the forthcoming help text property.